src/YawPitchControl/YawPitchControl.js

```diff
@@ -616,14 +616,19 @@ export default class YawPitchControl extends Component {
 	 *
 	 * @param {Object} coordinate yaw, pitch, fov
 	 * @param {Number} duration Animation duration. if it is above 0 then it's animated.
-	 */
+	 */	
 	lookAt({yaw, pitch, fov}, duration) {
+		const convToPositive = n=> (n+360*10)%360;
```

Comment:
I don't think this kind of approach is not the correct way to do this.
That's because the yaw range can be set arbitrarily, like -3600 to 3600 or so.
It's right that normally people won't set the value like that, but I think fine calculation for it would be nicer.